src/Compilers/CSharp/Portable/Generated/CSharpSyntaxGenerator/CSharpSyntaxGenerator.SourceGenerator/Syntax.xml.Internal.Generated.cs

```diff
@@ -30780,17 +30783,7 @@ public AttributeSyntax Attribute(NameSyntax name, AttributeArgumentListSyntax? a
         if (name == null) throw new ArgumentNullException(nameof(name));
 #endif
 
-        int hash;
-        var cached = CSharpSyntaxNodeCache.TryGetNode((int)SyntaxKind.Attribute, name, argumentList, this.context, out hash);
-        if (cached != null) return (AttributeSyntax)cached;
-
-        var result = new AttributeSyntax(SyntaxKind.Attribute, name, argumentList, this.context);
-        if (hash >= 0)
-        {
-            SyntaxNodeCache.AddNode(result, hash);
-        }
-
-        return result;
+        return new AttributeSyntax(SyntaxKind.Attribute, name, argumentList, this.context);
```

Comment:
there's a test that cacheable nodes hve no inheritable nodeflags set at all.  So if we want to maintain that invariant, attributes are no longer cacheable.